test/files/docWithAMQPSubscriber.yml

```
@@ -0,0 +1,83 @@
+asyncapi: '2.1.0'
```

Nitpick: Taking this opportunity to update it to the last (as per today) version: 2.3.0.
```suggestion
asyncapi: '2.3.0'
```